src/ctap/main_hid.rs

```diff
@@ -115,13 +112,13 @@ impl MainHid {
     }
 
     /// Returns whether a wink permission is currently granted.
-    pub fn should_wink(&self, now: CtapInstant) -> bool {
-        self.wink_permission.is_granted(now)
+    pub fn should_wink(&self) -> bool {
+        self.wink_permission.is_some() && self.wink_permission.unwrap().has_elapsed().is_some()
```

Comment:
Interesting. First I'm surprised this compiles. Second, the signature of `should_wink` might be problematic with the `has_elapsed` API. I see at least 2 options:
1. `should_wink` only returns the status since last update, i.e. `self.wink_permission.is_some()` (my preference)
2. `should_wink` is `&mut self` and also updates the timer.